actions/main/src/waitfordeployment.sh

```diff
@@ -3,14 +3,15 @@ waitfordeployment () {
   eval $(cat env_vars)
   if [[ "${google_zone}" != "" ]] && [[ "${K8_CLUSTER}" != "" ]]; then
     # wait for deployment to happen
-    sleep 5
+    echo "sleep 15 seconds to wait for ui pod to go down after helm deploy..."
+    sleep 15
     if kubectl get deployments -n ${NAMESPACEINUSE} ${NAMESPACEINUSE}-jade-datarepo-ui -o jsonpath="{.status}" | grep unavailable; then
       sleep 10
-      echo "pod unavailable waiting 10 seconds..."
+      echo "ui pod unavailable waiting 10 seconds..."
```

Comment:
I think you want to swap this with the line above (it prints after the sleep)

Reply:
In general, everything in `/actions/main` corresponds to our old actions and we probably don't want to spend time changing them except for high priority fixes (although your suggested change is straightforward)